src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 17 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

minishell for native executables.Streams from minishell.gets the error stream from minishell
```powershell
Expected strings to be the same, but they were different.
Expected length: 49
Actual length:   16
Strings differ at index 0.
Expected: 'Microsoft....'
But was:  'RuntimeException'
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 30
30:             $output.FullyQualifiedErrorId | Should -Be 'Microsoft.PowerShell.Commands.WriteErrorException'
```
Restart-Computer.Should restart the local computer
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1: line 28
28:             Restart-Computer -ErrorAction Stop | Should -BeNullOrEmpty

```
Restart-Computer.Restart-Computer Error Conditions.Should return the proper error when it occurs
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1: line 75
75:                 Restart-Computer -ErrorVariable RestartError 2> $null

```
Stop-Computer.Should stop the local computer
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Stop-Computer.Tests.ps1: line 29
29:             Stop-Computer -ErrorAction Stop | Should -BeNullOrEmpty

```
Stop-Computer.Stop-Computer Error Conditions.Should return the proper error when it occurs
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Stop-Computer.Tests.ps1: line 48
48:                 Stop-Computer -ErrorVariable StopError 2> $null

```